src/helper.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const getRandomDigit = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+const getArrayOfrandomValues = (length) => {
+  const result = [];
+  let newValue = getRandomDigit(0, 9);
+
+  while (result.length < length) {
+    if (!result.includes(newValue)) {
+      result.push(newValue);
+    }
+
+    newValue = getRandomDigit(0, 9);
+  }
+
+  return result;
+};
```

Comment:
```suggestion
  const result = [];

  for (let i = 0; i < length; ++i) {
    const newValue = getRandomDigit(0, 9);
    
    if (!result.includes(newValue)) {
      result.push(newValue);
    }
  }

  return result;
};
```

Reply:
This doesn't work as intended; in the case of generating an existing number, it will simply return an array with fewer elements.